packages/network/src/connection/WebRtcEndpoint.ts

```diff
@@ -117,6 +120,29 @@ export class WebRtcEndpoint extends EventEmitter implements IWebRtcEndpoint {
             .addQueriedMetric('messageQueueSize', () => {
                 return Object.values(this.connections).reduce((total, c) => total + c.getQueueSize(), 0)
             })
+
+        this.statusReportIntervalMs = 5 * 60 * 1000
+        this.startConnectionStatusReport()
+    }
+
+    private startConnectionStatusReport(): void {
+        if (this.statusReportTimeout) {
+            clearTimeout(this.statusReportTimeout)
+        }
+        this.statusReportTimeout = setTimeout(() => {
+            let connectedPeerCount = 0
+            const pendingConnectionIds = []
```

Comment:
Rename to `pendingPeerIds` as these the ids of `peers`, not ids of `connections` (https://github.com/streamr-dev/network-monorepo/blob/14e94a72c4b96a25f306e8e5526061cf8f7c2925/packages/network/src/connection/WebRtcConnection.ts#L125)